src/main/java/com/minecolonies/util/PermissionUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * todo: explain this class.
+ */
+public final class PermissionUtils
+{
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private PermissionUtils()
+    {
+    }
+
+    // todo: document!
+    @NotNull
+    public static List<Permissions.Player> getPlayersWithAtleastRank(@NotNull Colony colony, @NotNull Permissions.Rank rank)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)